widget/playground/src/containers/FunctionalLayout/FunctionalLayout.From.tsx

```diff
@@ -0,0 +1,40 @@
+import type { ChangeEvent } from 'react';
+
+import { Divider, TextField, Typography } from '@rango-dev/ui';
+import React from 'react';
+
+import { useConfigStore } from '../../store/config';
+import { DefaultChainAndToken } from '../DefaultChainAndToken';
+
+import { FromAmount, FromToContainer } from './FunctionalLayout.styles';
+
+export function FromSection() {
+  const {
+    config: { amount },
+    onChangeAmount,
+  } = useConfigStore();
+
+  const handleChangeAmount = (e: ChangeEvent<HTMLInputElement>) => {
+    onChangeAmount(parseFloat(e.target.value || '0'));
+  };
+
+  return (
+    <>
+      <FromToContainer>
```

Comment:
I think we need a padding similar to what we have done in widget for scrolled lists. (wallets, blockchains, ...) 
Also updated figma if doesn't match.

![Screenshot from 2023-10-21 12-41-34](https://github.com/rango-exchange/rango-client/assets/99180934/3c3d337b-cb10-4841-8d28-007d621555c4)

Reply:
In the radio list, we have the paddings. I'll fix it in the checkbox list in another PR about selectBlockchains because supportedWallets is refactored there.